.github/workflows/collector-builder.yml

```diff
@@ -71,7 +74,9 @@ jobs:
         id: builder-tag
         run: |
           COLLECTOR_BUILDER_TAG="${DEFAULT_BUILDER_TAG}"
-          if [[ "${{ github.event_name }}" == 'pull_request' ]]; then
+          if [[ "${{ github.event_name }}" == 'pull_request' || \
+                "${{ github.ref_type }}" == 'tag' || \
+                "${{ github.ref_name }}" =~ ^release- ]]; then
```

Comment:
Just out of curiosity, what would happen for a commit with both a tag and a release branch, it will build the same image twice?

Reply:
It will, but they will be tagged as different images. That was the case for these two runs:
- Tag: https://github.com/stackrox/collector/actions/runs/6455211309
- Release branch: https://github.com/stackrox/collector/actions/runs/6455167343

Reply:
Yeah, I see, at the end it boils down to ref_name vs make tag.